contrib/CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+# Contributing to electrs
```

Comment:
Please move to root-level (similar to https://github.com/rust-bitcoin/rust-bitcoin/blob/master/CONTRIBUTING.md)

Reply:
Sure